quickstarts/gradle/micronaut4/README.md

```diff
@@ -0,0 +1,46 @@
+---
+name: "Gradle :: Micronaut 4"
+description: Micronaut 4 application featuring REST endpoints (micronaut-http) with validation (micronaut-validation).
+---
+# Eclipse JKube Micronaut 4 quickstart
+
+Micronaut 4 application featuring REST endpoints (micronaut-http) with validation (micronaut-validation).
+
+## Requirements
+
+- JDK 17+
+- Kubernetes Cluster (Minikube, OpenShift, CRC, etc.)
+
+## Building and deploying the application
+
+```shell script
+# Kubernetes
+$ gradle clean build k8sBuild k8sResource k8sApply
+# OpenShift
+$ gradle clean build ocBuild ocResource ocApply
+```
+## Expected output
+
+Once you've deployed the application, you should be able to perform requests:
+
+### Minikube
+
+```shell script
+$ curl $(minikube ip):$(kubectl get svc micronaut-zero-config -n default -o jsonpath='{.spec.ports[].nodePort}')
+Hello from Micronaut deployed with JKube!
+$ curl $(minikube ip):$(kubectl get svc micronaut-zero-config -n default -o jsonpath='{.spec.ports[].nodePort}')/hello/world
+Hello word!
+$ curl $(minikube ip):$(kubectl get svc micronaut-zero-config -n default -o jsonpath='{.spec.ports[].nodePort}')/hello/1
+{"message":"name: size must be between 3 and 2147483647","_links":{"self":{"href":"/hello/1","templated":false}}}
+```
+
+### OpenShift
+
+```shell script
+$ curl $(oc get routes micronaut-zero-config -o jsonpath='{.spec.host}')
+Hello from Micronaut deployed with JKube!
+$ curl $(oc get routes micronaut-zero-config -o jsonpath='{.spec.host}')/hello/world
+Hello world!%
+$ curl $(oc get routes micronaut-zero-config -o jsonpath='{.spec.host}')/hello/1
+{"message":"name: size must be between 3 and 2147483647","_links":{"self":{"href":"/hello/1","templated":false}}}
```

Comment:
Have you tested this quickstart on [Red Hat OpenShift](https://developers.redhat.com/developer-sandbox)?

Reply:
No, I didn't, as you might see I didn't activate related checkbox.
```[] I tested my code in OpenShift```

But I will try to do it by using sandbox.
For k8s - I tested it for both Gradle and Maven.